Cargo.toml

```diff
@@ -40,8 +40,10 @@ encryption = ["dep:ring"]
 cloud = []
 # static bundling of dependencies
 bundled = ["rusqlite/bundled"]
-# use native CA roots, instead of bundled
-tls-native-roots = ["ureq/native-certs"]
+# use CA roots built into the library for all HTTPS access
+tls-webpki-roots = ["dep:hyper-rustls", "hyper-rustls/webpki-roots", "dep:reqwest", "reqwest/rustls-tls-webpki-roots"]
```

Comment:
To me, calling this "webpki" in contrast to "native" seems odd -- it's "web public key infrastructure" either way, right? -- and something like "tls-vendored-roots" or "tls-bundled-roots" would make more sense.